apps/web/src/composables/external.ts

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
External is not a vague name for the composable. Also see @casimir/exchange (in the common directory, also already used in the USDAmount component), and a consider extending that instead of adding this (using some of your logic from here there instead).